contracts/InterestRate/StableRate.sol

```diff
@@ -0,0 +1,248 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity 0.8.13;
+
+/**
+ * @title Logic for Venus stable rate.
+ */
+contract StableRateModel {
+    event NewStableInterestParams(
+        uint256 baseRatePerBlockForStable,
+        uint256 multiplierPerBlock,
+        uint256 multiplierPerBlockForStable,
+        uint256 jumpMultiplierPerBlockForStable,
+        uint256 kink,
+        uint256 stableRatePremium,
+        uint256 optimalStableLoanRate
+    );
+
+    /// @notice Indicator that this is an InterestRateModel contract (for inspection)
+    bool public constant isInterestRateModel = true;
+
+    uint256 private constant BASE = 1e18;
```

Comment:
Can we order entities in this contract per spec? I have been using solhint to help with this but I haven't turned on the rule yet because it will result in too many warnings /errors. We can discuss how we want to handle this more precisely